src/api/spec/controllers/webui/apidocs_controller_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe Webui::ApidocsController, type: :controller do
+  describe "GET index" do
+    context "correct setup" do
+      render_views
+
+      before do
+        get :index
+      end
+
+      it { expect(response.body).to have_title('Open Build Service') }
+    end
+
+    context "broken setup" do
+      let!(:old_location) { CONFIG['apidocs_location'] }
+
+      it "errors and redirects" do
+        CONFIG['apidocs_location'] = '/your/mom'
+
+        expect(Rails.logger).to receive(:error).with(
+          "Unable to load apidocs index file from #{CONFIG['apidocs_location']}. Did you create the apidocs?"
+        )
+
+        get :index
+
+        expect(flash[:error]).to eq("Unable to load API documentation.")
+        expect(response).to redirect_to(root_path)
+
+        CONFIG['apidocs_location'] = old_location
```

Comment:
Then I will also make sense to create a before hook with the code at line 19.

Like
```ruby
before do
  CONFIG['apidocs_location'] = '/your/mom'
end
```

What would it be better?
a) as it is
b) only to create the after hook
c) only to create the before hook
d) to create the after hook and the before hook